bentoml/tracing/__init__.py

```diff
@@ -12,64 +12,79 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# List of APIs for accessing remote or local yatai service via Python
-
 from contextlib import contextmanager
 
-from bentoml import config
-
+from dependency_injector.wiring import Provide, inject
 
-ZIPKIN_API_URL = config("tracing").get("zipkin_api_url")
-OPENTRACING_SERVER_ADDRESS = config("tracing").get("opentracing_server_address")
-OPENTRACING_SERVER_PORT = config("tracing").get("opentracing_server_port")
+from bentoml.configuration.containers import BentoMLContainer
 
 
+@inject
 @contextmanager
```

Comment:
```suggestion
@contextmanager
@inject
```

Reply:
I'm not sure if `inject` works on a context manager. It should works on a function at least.

Reply:
yes it works, since context manager just returns a function with the same signature, which can be injected